doc/appendix-me/appendix-me.md

```diff
@@ -0,0 +1,12 @@
+---
+slug: /poscreators/middleware-doc/montenegro
+title: Introduction
+---
+
+# Fiscalization in Montenegro
+This chapter expands on the information provided in the _General_ chapter by adding details specific to the fiscalization for Montenegro. This additional information is provided only where applicable. The remaining chapters, for which there is no extra information required, were omitted.
+
+Legal regulations and further information can be found [on the website of the Montenegran tax authority](https://upravaprihoda.gov.me/rubrike/aktuelnosti/232806/Sve-o-ELEKTRONSKOJ-FISKALIZACIJI-na-jednom-mjestu.html).
+
+
+<div class="alert alert--warning" role="alert">Please note that this information is only complete when combined with the <b>General</b> part. To implement the Middleware, users should get themselves familiar with the general information first and then refer to the country-specific details listed here.</div>
```

Comment:
```suggestion
---
slug: /poscreators/middleware-doc/montenegro
title: Introduction
---

# Fiscalization in Montenegro
This chapter expands on the information provided in the _General_ chapter by adding details specific to the fiscalization for Montenegro. This additional information is provided only where applicable. The remaining chapters, for which there is no extra information required, were omitted.

Legal regulations and further information can be found [on the website of the Montenegran tax authority](https://upravaprihoda.gov.me/rubrike/aktuelnosti/232806/Sve-o-ELEKTRONSKOJ-FISKALIZACIJI-na-jednom-mjestu.html).


<div class="alert alert--warning" role="alert">Please note that this information is only complete when combined with the <b>General</b> part. To implement the Middleware, users should get themselves familiar with the general information first and then refer to the country-specific details listed here.</div>
```

:::note

At the moment there is no fiskaltrust.Portal for the Montenegrin market. That means the Configuration of the Cashbox and the download of the Launcher have to be done manually. If you need help to getting started we are happy to assist you.

:::